llvm/include/llvm/CodeGen/TargetInstrInfo.h

```diff
@@ -2184,6 +2184,12 @@ class LLVM_ABI TargetInstrInfo : public MCInstrInfo {
                                        MachineBasicBlock::iterator &MIT,
                                        unsigned Flags) const;
 
+  /// Remove all Linker Optimization Hints associated with instructions in
+  // \p MIs and \return the number of hints removed.
+  virtual size_t clearLOHs(const SmallPtrSetImpl<MachineInstr *> &MIs) const {
```

Comment:
Avoid the TLA and explain how and why this would be used

Reply:
I'm guessing TLA is "three letter acronym"?